Flow.Launcher.Plugin/IPublicAPI.cs

```diff
@@ -165,33 +170,20 @@ public interface IPublicAPI
         void LogException(string className, string message, Exception e, [CallerMemberName] string methodName = "");
 
         /// <summary>
-        /// Load JsonStorage for current plugin. This is the method used to load settings from json in Flow
+        /// Load JsonStorage for current plugin's setting. This is the method used to load settings from json in Flow.
+        /// When the file is not exist, it will create a new instance for the specific type.
         /// </summary>
         /// <typeparam name="T">Type for deserialization</typeparam>
         /// <returns></returns>
-        T LoadJsonStorage<T>() where T : new();
+        T LoadSettingJsonStorage<T>() where T : new();
 
         /// <summary>
-        /// Save JsonStorage for current plugin. This is the method used to save settings to json in Flow.Launcher
+        /// Save JsonStorage for current plugin's setting. This is the method used to save settings to json in Flow.Launcher
         /// This method will save the original instance loaded with LoadJsonStorage.
+        /// This API call is for manually Save. Flow will automatically save all setting that has registered. 
```

Comment:
i think it will be better to also include how setting gets registered in the comment so plugin devs do not need to dig into code to find out.